eve/main.yml

```diff
@@ -7,10 +7,16 @@ branches:
 
 models:
   - env: &kind-env
+      WORKER_COUNT: '2'
       OPERATOR_REPO: git@github.com:scality/zenko-operator.git
       OPERATOR_IMAGE: '%(prop:OPERATOR_IMAGE_NAME)s:%(prop:OPERATOR_IMAGE_TAG)s'
       KIND_NODE_IMAGE: "kindest/node:v1.17.5@sha256:ab3f9e6ec5ad8840eeb1f76c89bb7948c77bbf76bcebe1a8b59790b8ae9a283a"
       VOLUME_ROOT: /artifacts
+      OIDC_REALM: "zenko"
+      OIDC_CLIENT_ID: "zenko-ui"
+      OIDC_USERNAME: 'zenko-end2end'
+      OIDC_ENDPOINT: 'ui.zenko.local'
```

Comment:
Can we call this endpoint: `UI_ENDPOINT` and set `OIDC_ENDPOINT:"keycloak.zenko.local"`?